source/components/organisms/Step/Step.tsx

```diff
@@ -37,19 +40,46 @@ interface DialogText {
   title: string;
   body: string;
 }
-const dialogText: Record<DIALOG_TEMPLATE, DialogText> = {
-  [DIALOG_TEMPLATE.MAIN_STEP_COMPLETIONS]: {
-    title: "Vill du stänga formuläret?",
-    body: "Formuläret sparas och du kan fortsätta fylla i det fram till sista dagen för inlämning.",
-  },
-  [DIALOG_TEMPLATE.MAIN_STEP]: {
-    title: "Vill du avbryta ansökan",
-    body: "Ansökan sparas i 3 dagar. Efter det raderas den och du får starta en ny.",
-  },
-  [DIALOG_TEMPLATE.SUB_STEP]: {
-    title: "Vill du stänga fönster utan att spara inmatad uppgift?",
-    body: "",
-  },
+
+const getDialogText = (template: DIALOG_TEMPLATE): DialogText => {
```

Comment:
Similarly to how other data is treated, could we refactor this into a map? That way we avoid switch cases that can be hard to follow, and separate data better.

```typescript
const dialogTextCloseInThreeDays = {
  title: "Vill du avbryta ansökan",
  body: "Ansökan sparas i 3 dagar. Efter det raderas den och du får starta en ny.",
};

const dialogTexts: Record<DIALOG_TEMPLATE, DialogText> = {
    [DIALOG_TEMPLATE.MAIN_STEP]: dialogTextCloseInThreeDays,
    [DIALOG_TEMPLATE.NEW_APPLICATION_STEP]: dialogTextCloseInThreeDays,
    [DIALOG_TEMPLATE.MAIN_STEP_COMPLETIONS]: {
      title: "Vill du stänga formuläret?",
      body: "Formuläret sparas och du kan fortsätta fylla i det fram till sista dagen för inlämning.",
    }
    ...
}

function getDialogText(template: DIALOG_TEMPLATE): DialogText {
  return dialogTexts[template] ?? { title: "", body: "" }
}
```

I realize now while writing that since some `DIALOG_TEMPLATE` share the same `DialogText` it might be _a bit_ untidy to move into `dialogTextCloseInThreeDays` etc. but I still think it's cleaner. Thoughts?